python/mxnet/callback.py

```diff
@@ -37,19 +37,34 @@ def _callback(iter_no, sym=None, arg=None, aux=None):
 
 
 def do_checkpoint(prefix, period=1):
-    """Callback to checkpoint the model to prefix every epoch.
+    """A callback that saves a model checkpoint every few epochs. Checkpoint is binary files
```

Comment:
Suggest rewording as:

A callback that saves a model checkpoint every few epochs. Each checkpoint is made up of couple of binary files: a model description file and a parameters (weights and biases) file. The model description file is named `prefix`--symbol.json and parameters file is named `prefix`-`epoch_number`.params